ios/MattermostShare/ShareViewController.m

```diff
@@ -56,12 +58,23 @@ - (void)viewDidLoad {
 
     if (isBackgroundUpload) {
       NSString *requestWithGroup = [NSString stringWithFormat:@"%@|%@", requestId, appGroupId];
+      
+      [SessionManager sharedSession].closeExtension = ^{
+        [extensionContext completeRequestReturningItems:nil
+                                      completionHandler:nil];
+        NSLog(@"Extension closed");
+      };
+      
+      [SessionManager sharedSession].sendShareEvent = ^(NSString* eventName) {
+        NSLog(@"Send Share Extension Event to JS");
+        [_bridge enqueueJSCall:@"RCTDeviceEventEmitter"
+                       method:@"emit"
```

Comment:
nit: I'm guessing there's no `go fmt` equivalent to enforce these things, but should the colons all line up here?

Reply:
done by the XCode IDE so I would guess is fine? I do not know the answer to this